styles/main.css

```
@@ -58,24 +66,34 @@ button.primary {
+
 .pretty-search-bar {
-display: flex;
+  display: flex;
   align-items: center;
   justify-content: space-between;
-
-  width: 480px; 
-  flex-shrink: 0;
-
+  
+  width: 90%;
```

Stylelint (17.6.0)

[error] Unexpected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)